NOTICE-binary

```diff
@@ -8035,6 +8035,7 @@ See bundled license files for details
 - com.google.protobuf:protobuf-java:3.7.1
 - com.google.protobuf:protobuf-java-util:3.7.1
 - com.google.auth:google-auth-library-credentials:0.13.0
+- cloudpickle:1.2.2
```

Comment:
As the cloudpicke is part of the source distribution, we also need to add cloudpickle in the NOTICE file in the project root directory